hardware/arduino/avr/cores/arduino/USBCore.cpp

```diff
@@ -445,15 +431,12 @@ int USB_RecvControl(void* d, int len)
 
 int SendInterfaces()
 {
-	int total = 0;
 	u8 interfaces = 0;
 
-#ifdef CDC_ENABLED
-	total = CDC_GetInterface(&interfaces);
-#endif
+	CDC_GetInterface(&interfaces);
 
-#ifdef HID_ENABLED
-	total += HID_GetInterface(&interfaces);
+#ifdef PLUGGABLE_USB_ENABLED
+	PUSB_GetInterface(&interfaces);
 #endif
 
 	return interfaces;
```

Comment:
it does't return 0 because `interfaces` is incremented by the callee (instead, the original `total` variable was completely useles)